src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -31,6 +31,7 @@ def setUpClass(cls):
         config = ConfigTests()
         cls.maxDiff = None
         cls.am = AppManager()
+        cls.am.reload()  # class uses non-mocked data
```

Comment:
Used a `reload()` in another class which swapped out the live data for mocked data. Rectifying here